samples/logs.js

```diff
@@ -57,7 +57,7 @@ async function writeLogEntry(logName) {
     // Asynchronously batch write the log entries
     await log.write([text_entry, json_Entry]);
 
-    // Asynchronously let the logging library dispatch logs
+    // Let the logging library dispatch logs synchronously
```

Comment:
Based on fact that async/await pattern is Promise based, when `await` is not used with an async function call the function will execute synchronously (see some explanation [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function)). If function itself uses async calls with `await` keyword, function will execute asynchronously (see more insights [here](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Async_await))